packages/vendure-plugin-customer-managed-groups/src/customer-managed-groups.service.ts

```diff
@@ -364,4 +371,141 @@ export class CustomerManagedGroupsService {
       isGroupAdministrator,
     };
   }
+
+  async updateGroupMember(
+    ctx: RequestContext,
+    input: UpdateCustomerManagedGroupMemberInput
+  ): Promise<CustomerManagedGroup> {
+    if (
+      !input.title &&
+      !input.firstName &&
+      !input.lastName &&
+      !input.emailAddress
+    ) {
+      throw new UserInputError(`empty body`);
+    }
+    const myGroup = await this.myCustomerManagedGroup(ctx);
+    if (myGroup) {
```

Comment:
To avoid too much nesting of `if/else` statements, it's better to handle `return`'s or `throw error`'s first. Example:

```
if (!myGroup) {
         throw new UserInputError(
        `no customer managed group exists for authenticated customer`
      );
}
// Now we don't need an `else` here, because the function ends when the error is thrown.
```